pkg/archive/archive_unix_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	"testing"
```

Comment:
> difference is unix.Stat_t doesn't have additional padding at the end

Oh, I see it now, they looked so similar 😊 

> I could probably remove all of the type assertions looking for syscall.Stat_t

It's fine to keep it as-is for this PR. Thanks for explaining!